packages/itwin/measure-tools/src/measurements/AngleMeasurement.ts

```diff
@@ -43,6 +43,13 @@ export interface AngleMeasurementProps extends MeasurementProps {
   startPoint?: XYZProps;
   center?: XYZProps;
   endPoint?: XYZProps;
+  formatting?: AngleMeasurementFormattingProps;
+}
+
+/** Formatting properties for angle measurement. */
+export interface AngleMeasurementFormattingProps {
+  /** Defaults to "AecUnits.ANGLE" and "Units.RAD" */
```

Comment:
Is there a reason that all the angle formats in this PR default to radians? These are designed to be displayed to people right? Wouldn't most people prefer degrees?